norm/arrayNormalization.pl

```diff
@@ -249,9 +249,17 @@ sub makeArraysToAssaysToFiles {
 			next;
 		}
 
-		# Get technology from Array design file using peach API. 
-		my $adfInfoUrl = $atlasSiteConfig->get_arrayexpress_adf_info_url;
-		my $arrayDataTech=`curl -s $adfInfoUrl$arrayDesign`;
+		# Get technology from Array design file using biostudies API. 
+		my $arrayDataTech = "";
+		if($exptAccession eq "E-CURD-50" || $exptAccession eq "E-CURD-51"){
+   			print("BioStudies E-MTAB-800 was split into E-CURD-50 and E-CURD-51 \n");
+			$arrayDataTech=`curl -s https://www.ebi.ac.uk/biostudies/api/v1/studies/E-MTAB-800`;
+		}elsif($exptAccession eq "E-CURD-61" || $exptAccession eq "E-CURD-63"){
+			print("Not included in bioStudies \n");
+			$arrayDataTech="Affymetrix";
```

Comment:
same as above https://github.com/ebi-gene-expression-group/atlas-analysis/pull/9/files#r1277556568

Reply:
I understand your point.  However, Atlas is not supporting the addition of microarray data anymore. 

I'll open an issue - so we can address this separately.